README.md

```diff
@@ -1,10 +1,33 @@
 # Minimum GHC Installer [![Build Status](https://img.shields.io/travis/fpco/minghc.svg?style=flat)](https://travis-ci.org/fpco/minghc)
 
+## Wait, are you sure you don't want `stack`?
```

Comment:
I `codified` `stack` to minimize ambiguity for someone very new to the Haskell ecosystem.